webapp/channels/src/components/user_group_popover/group_member_list/group_member_list.tsx

```diff
@@ -197,9 +195,8 @@ const GroupMemberList = (props: Props) => {
                                     {user: name})}
                                 onClick={() => showDirectChannel(user)}
                             >
-                                <LocalizedIcon
+                                <i
```

Comment:
nit: in this case everything can be in one line because it only has 1 attribute.

Reply:
I will keep that in mind